### PR TITLE
Update MQTT and AMQP minor package versions

### PR DESCRIPTION
**Pull request by drwill-ms**

### CODE REVIEW

**Review by abhipsaMisra — APPROVED** [review]

**Review by patilsnr — APPROVED** [review]

File: e2e/Tests/helpers/templates/FaultInjection.cs [hunk 1]
```diff
@@ -78,15 +79,9 @@ internal static bool FaultShouldDisconnect(string faultType)
 
                 await deviceClient.SendTelemetryAsync(faultInjectionMessage, linkedCts.Token).ConfigureAwait(false);
             }
-            catch (IotHubClientException ex) when (ex.IsTransient || ex.ErrorCode == IotHubClientErrorCode.ConnectionForcefullyClosedOnFaultInjection)
```
Comment (drwill-ms) [inline e2e/Tests/helpers/templates/FaultInjection.cs:81]: This was showing up in GWv2, but they said they'd roll it back, so we don't need it anymore.

File: e2e/Tests/helpers/templates/FaultInjection.cs [hunk 2]
```diff
@@ -78,15 +79,9 @@ internal static bool FaultShouldDisconnect(string faultType)
...
+            catch (IotHubClientException ex) when (ex.IsTransient || ex.InnerException is MqttClientDisconnectedException)
             {
                 VerboseTestLogger.WriteLine($"{nameof(ActivateFaultInjectionAsync)}: {ex}");
-
-                // For quota injection, the fault is only seen for the original HTTP request.
-                if (transportSettings is IotHubClientHttpSettings)
```
Comment (drwill-ms) [inline e2e/Tests/helpers/templates/FaultInjection.cs:86]: Irrelevant in v2.

File: iothub/device/src/Pipeline/MqttTransportHandler.cs [hunk 3]
```diff
@@ -439,9 +439,9 @@ public override async Task SendTelemetryAsync(TelemetryMessage message, Cancella
 
                 if (message.Properties.ContainsKey("AzIoTHub_FaultOperationType"))
                 {
-                    // When fualt injection causes this operation to fail, the MQTT layer throws a TaskCanceledException.
+                    // When fault injection causes this operation to fail, the MQTT layer throws a MqttClientDisconnectedException.
```
Comment (drwill-ms) [inline iothub/device/src/Pipeline/MqttTransportHandler.cs:442]: This was the breaking change from the MQTT lib.

**Pull request merged**